src/navigator.js

```diff
@@ -271,6 +273,31 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
             }
         }
     },
+    /**
+      /* Flip navigator element
+      */
+    toogleFlip: function() {
+      this.doFlip = !this.doFlip;
+
+      this.setDisplayTransform(this.viewer.doFlip ? "scale(-1,1)" : "scale(1,1)");
+      this.viewport.viewer.forceRedraw();
+    },
+
+    setDisplayTransform: function(rule) {
+      this.displayRegion.style.webkitTransform = rule;
```

Comment:
This can be `setElementTransform(this.displayRegion, rule;`. Same for this.canvas.

Reply:
I tried to avoid that approach since it woudn't be used anymore in the code but yes, I thought about that. Do you really thing it is a good practice and that it should be done here anyway?

Reply:
Done.